functions/src/index.ts

```diff
@@ -231,4 +233,22 @@ app.post('/isVerified', isVerified)
  */
 app.post('/sendVerificationEmail', sendVerificationEmail)
 
+/**
+ * API for getting the submissions for each problem in a given list of problems
+ * @req JSON with the user id, the list of problemIds needed, and isBrief which
```

Comment:
I might just not be too familiar with this. What are the data types of these IDs? I see in your screenshot, you have a "-1" as well as a -1. Are they equivalent? Also does -1 refer to a custom (non-problem) submission? Would an example for a problem ID for problem submissions simply be like "twosum"? Or is there some other hashed ID?

Reply:
We're using -1 to refer to a non-problem submission.

Reply:
The ID can be a number or a string. Firebase doesn't care about types a field are. Two documents in the same collection can be entirely different and firebase wouldn't care. This means we can have -1 as a number for the problem ID of an arbitrary submission and a string problem ID for a problem such as "twosum". This was also tested in my API with -1, "-1" and "-2". I think the type distinction between an arbitrary submission and a problem submission is important because now if someone wants to, they can create a problem called "-1" with the same problem ID.

Reply:
Thanks for the clarification!